### PR TITLE
add table-progress in components

### PR DESCRIPTION
**Pull request by JaswanthReddyPadala**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [x] 🚩 Other

## Description
Added a progress table component in components.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
This is the new progress table component added. ✨
Screenshot 2021-10-01 100722

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
Awesome Components

### CODE REVIEW

File: components/progress-table.html [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+                        hover:bg-gray-100
+                        border-b border-t border-gray-100
+                    "
+                >
+                    <td class="pl-4 cursor-pointer">
+                        <div class="flex items-center">
+                            <div class="w-10 h-10">
+                                <img
+                                    class="w-full h-full"
+                                    src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.chQQuCzLQ4NbSGSmu9OBEgEwFS%26pid%3DApi&f=1"
```
Comment (thinkverse) [inline components/progress-table.html]: Please change these images to something from Unsplash instead. 👍

**Review by thinkverse — CHANGES_REQUESTED** [review]
If you could reformat your code so the classes and SVG attributes aren't on new lines that would be great. 👍

File: components/progress-table.html (hunk 1 above)
Comment (JaswanthReddyPadala) [reply components/progress-table.html]: fixed image links

Comment (thinkverse) [reply components/progress-table.html]: If you could also fix the formatting on the CSS classes and SVG attributes this will be good to merge. 👍

Comment (JaswanthReddyPadala) [reply components/progress-table.html]: yes, I have formated CSS and SVG attributes properly, Hope it is fine now.

File: components/progress-table.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
+<div class="w-full sm:px-6">
+    <div class="px-4md:px-10py-4md:py-7bg-gray-100rounded-tl-lg rounded-tr-lg">
```
Comment (thinkverse) [inline components/progress-table.html]:
```suggestion
<div class="w-full h-screen bg-white sm:px-6">
    <div class="px-4 md:px-10 py-4 md:py-7 bg-gray-100 rounded-tl-lg rounded-tr-lg">
```

**Review by thinkverse — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**